app/frontend/components/domains/single-zone-cooling-heating-tool/single-zone-cooling-heating-tool-list-screen.tsx

```diff
@@ -0,0 +1,220 @@
+import {
+  Button,
+  Container,
+  Heading,
+  HStack,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Spinner,
+  Tab,
+  Table,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+  useDisclosure,
+} from "@chakra-ui/react"
+import { observer } from "mobx-react-lite"
+import React, { useEffect, useState } from "react"
+import { FormProvider, useForm } from "react-hook-form"
+import { useTranslation } from "react-i18next"
+import { useMst } from "../../../setup/root"
+import { CoverSheetForm } from "./cover-sheet-form"
+import { InputSummaryForm } from "./input-summary-form"
+import { RoomByRoomForm } from "./room-by-room-form"
+
+export const SingleZoneCoolingHeatingToolListScreen = observer(() => {
+  const { t } = useTranslation() as any
+  const { pdfFormStore } = useMst()
+  const formMethods = useForm()
+  const [activeTabIndex, setActiveTabIndex] = useState(0)
+  const [loadingPdfs, setLoadingPdfs] = useState<Record<string, boolean>>({})
+  const [generatedPdfs, setGeneratedPdfs] = useState<Record<string, string>>({})
+  const [editingId, setEditingId] = useState<string | null>(null)
+  const [editTabIndex, setEditTabIndex] = useState(0)
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
+  useEffect(() => {
+    pdfFormStore.getPdfForms()
+  }, [])
+
+  const pdfForms = pdfFormStore.pdfForms
+
+  const generatePdf = async (id: string) => {
+    setLoadingPdfs((prev) => ({ ...prev, [id]: true }))
+
+    try {
+      const result = await pdfFormStore.generatePdf(id)
+
+      setTimeout(() => {
+        setLoadingPdfs((prev) => ({ ...prev, [id]: false }))
+
+        setGeneratedPdfs((prev) => ({ ...prev, [id]: "ready" }))
+      }, 5000)
+    } catch (error) {
+      console.error("Error generating PDF:", error)
+      setLoadingPdfs((prev) => ({ ...prev, [id]: false }))
+    }
+  }
+
+  const handleDownloadPdf = async (id: string) => {
```

Comment:
the may be replicated by downloadFileFromStorage
see app/frontend/utils/utility-functions.ts